tasks/db_deploy/orca_sql.py

```diff
@@ -369,16 +369,17 @@ def recovery_file_table_sql() -> TextClause:
         -- Create table
         CREATE TABLE IF NOT EXISTS recovery_file
         (
-          job_id              text NOT NULL
-        , granule_id          text NOT NULL
-        , filename            text NOT NULL
-        , key_path            text NOT NULL
-        , restore_destination text NOT NULL
-        , status_id           int2 NOT NULL
-        , error_message       text NULL
-        , request_time        timestamp with time zone NOT NULL
-        , last_update         timestamp with time zone NOT NULL
-        , completion_time     timestamp with time zone NULL
+          job_id                 text NOT NULL
+        , granule_id             text NOT NULL
+        , filename               text NOT NULL
+        , key_path               text NOT NULL
+        , multipart_chunksize_mb numeric NULL
```

Comment:
Do we need to also update the recovery schema to show the new column `multipart_chunksize_mb `

Reply:
ads_orca_schema.xed? Took my best guess, but not sure how to properly edit.

Reply:
this one `https://github.com/nasa/cumulus-orca/blob/develop/website/static/files/ads_orca_schema_refined.xed`
I opened this file and edited in aqua data studio and then uploaded the file here.

Reply:
Thank you. Fixed.